oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticDocumentMaker.java

```diff
@@ -99,23 +99,32 @@ protected void indexAnalyzedProperty(ElasticDocument doc, String pname, String v
 
     @Override
     protected void indexSuggestValue(ElasticDocument doc, String value) {
-        doc.addSuggest(value);
+        if (value != null && value.length() > 0) {
```

Comment:
Some would say "!value.isEmpty()" is better... but I would actually keep it like it is, because of the "!"